src/controller/python/test/test_scripts/network_commissioning.py

```diff
@@ -143,7 +143,7 @@ async def test_wifi(self, endpointId):
         logger.info(f"Scan networks")
         req = Clusters.NetworkCommissioning.Commands.ScanNetworks(
             ssid=b'', breadcrumb=self.with_breadcrumb())
-        res = await self._devCtrl.SendCommand(nodeid=self._nodeid, endpoint=endpointId, payload=req)
+        res = await self._devCtrl.SendCommand(nodeid=self._nodeid, endpoint=endpointId, payload=req, responseType=None, timedRequestTimeoutMs=None, interactionTimeoutMs=20000)
```

Comment:
this modification happens in wifi, but we are testing thread provisioning, please modify scan network in test_thread function https://github.com/turon/connectedhomeip/blob/51a8491dae8968ffc318da5091550809efb8bbcf/src/controller/python/test/test_scripts/network_commissioning.py#L280.

Also ConnectNetwork is also async behavior, which may also need more time than default mrp interval

Reply:
We should read scan/connect timeout parameter attribute for WiFi/thread network cluster, which should be defined in spec, then set it up for scan/connect command. @erjiaqing, please prioritize this work to make this behavior happen in test.

In the past， since MRP was enough big, WiFi/thread scan/connect may be fine without customized timeout. With this change, we should make sure both chip-tool and python tool take right timeout for WiFi/thread scan/connect command during verification test

Reply:
Added explicit interactionTimeout to tests involving long IM operations: network scan, multiple subscriptions, etc.